docs/server/installation.rst

```diff
@@ -124,10 +79,7 @@ To activate the virtual environment run the :command:`activate` script:
 
   $ source env/bin/activate
```

Comment:
I am used to `workon` provided by `virtualenvwrapper`, but I suppose we can live without that.